source/templates.tex

```diff
@@ -2800,24 +2800,26 @@
 \end{example}
 
 \pnum
-The instantiation of a pack expansion
-that is neither a \tcode{sizeof...} expression
-nor a \grammarterm{fold-expression}
-produces a
-list of elements
-$\mathtt{E}_1,$ $\mathtt{E}_2,$ $\cdots,$ $\mathtt{E}_N$,
+The instantiation of a pack expansion considers
+items $\mathtt{E}_1,$ $\mathtt{E}_2,$ $\cdots,$ $\mathtt{E}_N$,
```

Comment:
a) should this be `$\tcode{E}_1, \tcode{E}_2` etc?

b) should this be `\dotsc`, not `cdots`?

Reply:
(I could do this separately if you prefer to keep the diff small, just as  a note-to-self then.)

Reply:
Style Guidelines:
"Use \cdots when \dotsc (or \ldots) could be confused with a pack expansion."

Updated to \dotsc and fixed the math mode.

@tkoeppe, this seems ready now.

Reply:
Ah, I see. Was there a risk of confusion? It seems pretty clear.